components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -786,33 +659,46 @@ public void testChown(boolean isSudoing, boolean permChown,
         if (isSudoing) {
             Assert.assertFalse(getCurrentPermissions(image).canChown());
             doChange(client, factory, Requests.chown().target(image).toUser(anotherUser.userId).build(), false);
-            long imageGroupId = ((RLong) iQuery.projection(
-                    "SELECT details.group.id FROM Image i WHERE i.id = :id",
-                    new ParametersI().addId(image.getId())).get(0).get(0)).getValue();
-            Assert.assertEquals(imageGroupId, normalUser.groupId);
             assertOwnedBy(image, normalUser);
-            assertOwnedBy((new OriginalFileI(remoteFileId, false)), normalUser);
-        } else {
+            assertOwnedBy(originalFile, normalUser);
+            assertOwnedBy(annotOriginalFileAnnotationTagAndLinks, normalUser);
+        } else if (chownPassingWhenNotSudoing && annotationsChownExpectSuccess) {
             /* chowning the image NOT being sudoed,
              * should pass only in case you have Chown
              * privilege, captured in "chownPassingWhenNotSudoing" boolean.
              * Also check that the value of canChown boolean matches chownPassingWhenNotSudoing
              * boolean in such case.*/
-            Assert.assertEquals(getCurrentPermissions(image).canChown(), chownPassingWhenNotSudoing);
+            Assert.assertEquals(getCurrentPermissions(image).canChown(), true);
```

Comment:
Fixed in another prepared PR where the line https://github.com/pwalczysko/openmicroscopy/commit/04efe7400c43be1073ef47ee97356683a906ff0d#diff-d91232ef1e245a87e311c4a79a2a5f77L669 is replaced by https://github.com/pwalczysko/openmicroscopy/commit/04efe7400c43be1073ef47ee97356683a906ff0d#diff-d91232ef1e245a87e311c4a79a2a5f77R655